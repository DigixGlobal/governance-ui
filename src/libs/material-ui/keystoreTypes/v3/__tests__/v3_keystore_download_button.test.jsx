import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import V3KeystoreDownloadButton from '../v3_keystore_download_button';

const keystore = {
  id: 1,
  data: {
    address: '0x1435345',
  },
  addresses: [{
    address: '0x12345',
  }, {
    address: '0x1234567',
  }],
};

describe('<V3KeystoreDownloadButton />', () => {
  test('renders correctly', () => {
    const component = shallow(
      <V3KeystoreDownloadButton
        keystore={keystore}
      />,
    );

    expect(toJson(component)).toMatchSnapshot();
  });
});
