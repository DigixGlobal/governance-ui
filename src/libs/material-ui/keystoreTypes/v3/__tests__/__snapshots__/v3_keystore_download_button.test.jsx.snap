// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<V3KeystoreDownloadButton /> renders correctly 1`] = `
<DownloadJSONButton
  getFileInfo={[Function]}
  keystore={
    Object {
      "addresses": Array [
        Object {
          "address": "0x12345",
        },
        Object {
          "address": "0x1234567",
        },
      ],
      "data": Object {
        "address": "0x1435345",
      },
      "id": 1,
    }
  }
  props={Object {}}
/>
`;
